helm-chart/splunk-operator/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: splunk-operator
+description: A Helm chart for the Splunk Operator for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.3
```

Comment:
not sure of the version should we just start with 1.0.0

Reply:
Changed it to 1.0.0, needs to be incremented for the Chart Releaser tool when updating the repo.